nimare/annotate/gclda.py

```diff
@@ -405,14 +635,85 @@ def _update(self, loglikely_freq=1):
             is 1 (log-likelihood is updated every iteration).
         """
         self.iter += 1  # Update total iteration count
-
         LGR.debug(f"Iter {self.iter:04d}: Sampling z")
         self.seed += 1
-        self._update_word_topic_assignments(self.seed)  # Update z-assignments
+        (
+            self.topics["wtoken_topic_idx"],
+            self.topics["n_word_tokens_word_by_topic"],
+            self.topics["total_n_word_tokens_by_topic"],
+            self.topics["n_word_tokens_doc_by_topic"],
+        ) = _update_word_topic_assignments(
+            self.topics["wtoken_topic_idx"],
+            self.topics["n_word_tokens_word_by_topic"],
+            self.topics["total_n_word_tokens_by_topic"],
+            self.topics["n_word_tokens_doc_by_topic"],
+            self.topics["n_peak_tokens_doc_by_topic"],
+            np.array(self.data["wtoken_doc_idx"]),
+            np.array(self.data["wtoken_word_idx"]),
+            len(self.vocabulary),
+            self.params["beta"],
+            self.params["gamma"],
+            self.seed,
```

Comment:
Probably that explains the difference you saw because we are testing with a small number of iterations. I think `self.seed = self.params["seed_init"]` is used to initialize the `peak_topic` and the `peak_subregion` assignments, whereas `self.seed` is used to perform the sampling in `_update()`.